snippets/xml_snippets.json

```diff
@@ -418,7 +418,7 @@
     "Field tag": {
         "prefix": "oofield",
         "body": [
-            "<field name=\"${1}\"></field>"
+            "<field name=\"${1}\"${20}${30:</field}>${50}"
```

Comment:
Should be `<field name=\"${1}\"${2|></field>,/>|}`

Reply:
Hmm the problem is that then you cannot set a tabstop in the middle of the opening tag and the closing tag. That's a very useful tab space if you really need a closing tag.

Let me just use 2 different definitions as you suggested before.